etcdadm/etcdadm

```diff
@@ -332,9 +332,9 @@ _docker_awscli_command() {
   dir=$(dirname "$(config_state_dir)")
   echo docker
   echo run
+  echo --rm
   printf "%s\n" "-e"
   echo "AWS_DEFAULT_REGION=$aws_region"
-  echo "--rm"
```

Comment:
@ytsarev Thanks for the confirmation 👍
I tried to reproduce the issue by running `docker run -e FOO=BAR --rm ...` and `docker run --rm -e FOO=BAR` on my docker v1.11.1 running in minikubeVM but had no luck so far.
Probably there may be unknown variable(s) other than the order of command-line args causing the issue.